process/io/plot_proc.py

```diff
@@ -10520,16 +10519,18 @@ def plot_plasma_pressure_gradient_profiles(axis, mfile_data, scan):
         mfile_data.data[f"pres_plasma_ion_total_profile{i}"].get_scan(scan)
         for i in range(n_plasma_profile_elements)
     ]
+    pres_plasma_profile_total = [
+        mfile_data.data[f"pres_plasma_total_profile{i}"].get_scan(scan)
+        for i in range(n_plasma_profile_elements)
+    ]
```

Comment:
Seems like a fair bit of duplication for the 4 profiles now. Could you put all of this into a single function or loop?